index.js

```diff
@@ -69,11 +69,11 @@ const botometer = function(config) {
   }
 
   // returns a user object, their latest tweets and mentions, and bot score
-  this.getBotScore = function(screen_name) {
-    const data = {user:null,timeline:null,mentions:null};
+  this.getBotScore = function({ screenName, userId }) {
+    const data = { user:null, timeline:null, mentions:null };
     return new Promise((resolve, reject) => {
       // get this user's timeline - latest 200 tweets
-      this.searchTwitter('statuses/user_timeline',{screen_name:screen_name,count:200})
+      this.searchTwitter('statuses/user_timeline', { screen_name:screenName, user_id:userId, count:200 })
```

Comment:
What's going on if the userId is `undefined`? Will Twitter send you a response with user data?

Reply:
Yes, only one of the two needs to be defined.